test/karma/launchers.js

```diff
@@ -211,77 +211,96 @@ module.exports = {
 		os_version: "High Sierra"
 	},
 
-	"bs_ios-5.1": {
+	"bs_iphone_7-10.3": {
```

Comment:
Sizzle supports iOS 5.1+ so don't remove older devices. Also, we don't need two copies of the same iOS version like 10.3 here.

Reply:
OK, so we can't really add old Android versions back as they're too unstable (and will be removed in a few months) but please only leave one entry for each iOS version and name them like they were named currently, i.e. `bs_ios_VERSION`.